components/org.wso2.carbon.identity.oauth.client.authn.filter/src/main/java/org/wso2/carbon/identity/oauth/client/authn/filter/OAuthClientAuthenticatorProxy.java

```diff
@@ -47,6 +48,8 @@ public class OAuthClientAuthenticatorProxy extends AbstractPhaseInterceptor<Mess
 
     private static final Log log = LogFactory.getLog(OAuthClientAuthenticatorProxy.class);
     private static final String HTTP_REQUEST = "HTTP.REQUEST";
+    private static final List<String> PROXY_ENDPOINT_LIST = Arrays.asList("/oauth2/token", "/oauth2/revoke",
+            "/oauth2/device_authorize", "/oauth2/ciba", "/oauth2/par");
```

Comment:
better if we can have this to be configurable

Reply:
Since these endpoints are highly unlikely to be modified or changed there is no significance need of making these endpoints configurable. Even though we introduce new endpoints we can easily add the respective endpoints to this variable rather than adding it to the configurations.